tests/rdt/transformers/test_CatTransformer.py

```diff
@@ -178,3 +179,20 @@ def test_get_probability_map(self):
             # Mean is middle point
             # We check this way because of floating point issues
             assert (mean - interval[0]) - (interval[1] - mean) < 1 / 1E9
+
+    def test_fit_transform_val_nan(self):
+        """Tests that nans are handled by fit_transform method"""
+
+        # Setup
+        data = pd.Series([np.nan, 1, 5])
+        col_meta = {
+            "name": "breakfast",
+            "type": "categorical"
+        }
+        transformer = CatTransformer()
+        # Run
+        output = transformer.fit_transform(data, col_meta)
+
+        # Check
+        # the  nan value in the data should be in probability map
+        assert output is not None
```

Comment:
This is not checking that the nan value is in the probability map.

The check for that should be something like 

```python
assert -1 in transformer.category_map
```
But in the concrete case, you are using integers as categorical data, and if that's a valid use case, then you can't guaratee that you won't be having a category with value -1.